packages/react-tweet/src/twitter-theme/tweet-quoted-tweet.tsx

```diff
@@ -0,0 +1,25 @@
+import { QuotedTweet } from '../api'
+import { TweetQuotedTweetHeader } from './tweet-quoted-tweet-header.js'
+import { TweetQuotedTweetBody } from './tweet-quoted-tweet-body.js'
+import { TweetQuotedTweetContainer } from './tweet-quoted-tweet-container.js'
+import { TweetQuotedTweetMedia } from './tweet-quoted-tweet-media.js'
+import { useMemo } from 'react'
+import { enrichQuotedTweet } from '../utils'
```

Comment:
When running the dev apps, the `create-react-app` app fails to build because it can't find this file to import I believe you'll want to import from `../utils.js` file here.